GNUmakefile

```
@@ -0,0 +1,49 @@
+NAME=image-builder
+ROOT_DIR:=$(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+BUILD_DIR=build
+PLUGIN_DIR=${BUILD_DIR}/plugins
+VERSION=$(shell git describe --tags --abbrev=0 | cut -b 2-)
+BINARY=packer-plugin-${NAME}_v${VERSION}_x5.0_linux_amd64
+# https://github.com/hashicorp/packer-plugin-sdk/issues/187
+HASHICORP_PACKER_PLUGIN_SDK_VERSION?="v0.5.2"
+PLUGIN_FQN=$(shell grep -E '^module' <go.mod | sed -E 's/module \s*//')
+PLUGIN_PATH=./cmd/plugin
+
```

I'm not overly happy about the syntax we use in other `Makefiles` of the projects but maybe you want to add this or a similar target?

```suggestion

.PHONY: help
help: ## print this help
	@echo "make [TARGETS...]"
	@echo
	@echo 'Targets:'
	@awk 'match($$0, /^([a-zA-Z_\/-]+):.*?## (.*)$$/, m) {printf "  \033[36m%-30s\033[0m %s\n", m[1], m[2]}' $(MAKEFILE_LIST) | sort
	
```